packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -136,22 +136,22 @@
 
   <Shw::Flex as |SF|>
     <SF.Item @label="Text only">
-      <ul class="shw-component-tabs-sample-ul-wrapper">
+      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
```

Comment:
@MelSumner to avoid conflicts I have done these changes directly in https://github.com/hashicorp/design-system/pull/1688

see commit: 0171d1b21c34c2e89a15edbdb2d9aedf8a373a86

Reply:
@MelSumner can you rebase this branch on `main` so I can review and approve? thanks

Reply:
@MelSumner OK, my bad I was confused: you need to wait for https://github.com/hashicorp/design-system/pull/1688 to be merged first. Sorry (too many open PRs) 🤦‍♂️

Reply:
OK, now the PR is merged, you can rebase. Sorry for the delay.